### PR TITLE
fix: adding CVE patch and patching instructions

### PR DESCRIPTION
**Pull request by shipperizer**
fixing CVE on main and added description on how to address future ones 

closing #448 and #449

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+```
+
+* let the machinery do its job 
+
+
+
+### after https://github.com/canonical/identity-platform-admin-ui/pull/452 merge
+
+
+In this case OCI tags don't inclue the patch version anymore, we should be able to simply sue the current workflows
```
Comment (nsklikas) [inline SECURITY.md:42]:
```suggestion
In this case OCI tags don't inclue the patch version anymore, we should be able to simply use the current workflows
```

**Review by nsklikas — COMMENTED** [review]
We already have some security docs in https://github.com/canonical/identity-platform-admin-ui/tree/main/docs. Should we place this in there, not sure what the "Canonical Standard" is

**Comment by shipperizer** [comment]
> We already have some security docs in https://github.com/canonical/identity-platform-admin-ui/tree/main/docs. Should we place this in there, not sure what the "Canonical Standard" is

it will be mandatory to have a security.md during this cycle on the root folder

File: SECURITY.md (hunk 1 above)
Comment (cjdcordeiro) [inline SECURITY.md:42]:
```suggestion
In this case OCI tags don't include the patch version anymore, we should be able to simply sue the current workflows
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+       - name: Release
+         run: |
+-          $OCI_FACTORY upload -y --release track=$IMAGE_VERSION_STABLE-22.04,risks=stable,eol=$EOL_STABLE
+           $OCI_FACTORY upload -y --release track=$IMAGE_VERSION_CANDIDATE-22.04,risks=candidate,edge,eol=$EOL_CANDIDATE
+         env:
+           GITHUB_TOKEN: ${{ secrets.token }} 
+```
+
+
+* apply CVE patches (conventional commits won't be trigger a release here, so using a chore/feat/fix won't make a difference)
```
Comment (zhijie-yang) [inline SECURITY.md:98]:
```suggestion
* apply CVE patches (conventional commits won't trigger a release here, so using a chore/feat/fix won't make a difference)
```

**Review by zhijie-yang — APPROVED** [review]
Looks good, just one nitpick here.

**Comment by pik4ez-canonical** [comment]
Newbie question. Is there a way to release security patches without overwriting existing tags?

**Review by pik4ez-canonical — APPROVED** [review]

**Review by pik4ez-canonical — APPROVED** [review]

**Review by wood-push-melon — DISMISSED** [review]

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+* checkout a new branch from the tag and name it `cve/<tag>`
+
+```git
+git switch --detach v1.19.0
+git switch -c cve/v1.19.0
+```
+
+* apply [oci-factory workflow patch](https://github.com/canonical/identity-platform-admin-ui/commit/eb0b7859f2210c9e2ce500e397ae3da688fef4de)
+* apply CVE patches (conventional commits won't trigger a release here, so using a chore/feat/fix won't make a difference)
+* retag to the head fo the branch and push the tag
```
Comment (BarcoMasile) [inline SECURITY.md:28]:
```suggestion
* retag to the head of the branch and push the tag
```

**Review by pik4ez-canonical — APPROVED** [review]

**Review by nsklikas — APPROVED** [review]

**Pull request merged**